packages/manager/modules/manager-pci-common/vite.config.mjs

```diff
@@ -0,0 +1,57 @@
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vitest/config';
+import dts from 'vite-plugin-dts';
+import path from 'path';
+import tailwindcss from 'tailwindcss';
+import { getBaseConfig } from '@ovh-ux/manager-vite-config';
+import { visualizer } from 'rollup-plugin-visualizer';
+import * as packageJson from './package.json';
+
+const baseConfig = getBaseConfig({});
+const pathSrc = path.resolve(__dirname, 'src');
+
+export default defineConfig({
+  ...baseConfig,
+  resolve: {
+    alias: {
+      '@/': pathSrc,
+    },
+  },
+  plugins: [
+    react(),
+    dts({
+      root: __dirname,
+      insertTypesEntry: true,
+      outDir: 'dist/types',
+    }),
+    visualizer({
+      filename: 'stats/bundle-analysis.html',
+    }),
+  ],
+  css: {
+    postcss: {
+      plugins: [tailwindcss],
+    },
+  },
+  test: {
+    globals: true,
+    environment: 'jsdom',
+  },
+  build: {
+    outDir: path.resolve(__dirname, 'dist'),
+    emptyOutDir: true,
+    lib: {
+      entry: path.resolve(pathSrc, 'index.ts'),
+      name: 'ManagerPciCommonLib',
+      fileName: 'index',
+      formats: ['es'],
+    },
+    rollupOptions: {
+      external: (id) => {
+        const deps = [...Object.keys(packageJson.peerDependencies || {})];
+        return deps.some((dep) => id === dep || id.startsWith(`${dep}/`));
+      },
```

Comment:
Since `rollup` considers exact path of imports, i think you should also addd `@ovhcloud/ods-components/react` to not include ODS components in the bundle.